_drafts/SC1-2-3+SC1-2-5-Video-description-track.md

```diff
@@ -0,0 +1,106 @@
+---
+name: Video element description track
+group: SC1-2-video-element
+description: |
+  This rule checks that non-streaming video elements with a description track have that track be descriptive.
+
+success_criterion:
+- 1.2.3 # Audio Description or Media Alternative (Prerecorded)
+- 1.2.5 # Audio Description (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] video element [visible in the page][] where the video contains [audio][] and a `track` element with a `kind="description"` attribute.
```

Comment:
"visible on the page" not "in the page" ehttps://github.com/auto-wcag/auto-wcag/blob/master/pages/algorithms/visible-on-the-page.md

Reply:
Thanks!